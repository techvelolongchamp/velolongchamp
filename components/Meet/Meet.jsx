import React from 'react'
import { FaMapMarkedAlt } from 'react-icons/fa'
import { useIntl } from 'react-intl'

import { AgendaContainer } from './Meet.styled'

const Meet = () => {
  const { formatMessage } = useIntl()
  const displayDate = () => {
    return formatMessage({ id: 'meetUs.noMeeting' })
  }

  return (
    <AgendaContainer>
      <p>{formatMessage({ id: 'meetUs.message1' })}</p>
      <p>
        {formatMessage({ id: 'meetUs.message2' })}
        <a
          href="https://goo.gl/maps/2sdy3M1yRUho9k8G9"
          target="_blank"
          rel="noopener noreferrer"
          title="RDV"
        >
          <FaMapMarkedAlt />
        </a>
      </p>
      <p>{formatMessage({ id: 'meetUs.message3' })}</p>
      <p>{formatMessage({ id: 'meetUs.message4' })}</p>
      <ul>{displayDate()}</ul>
    </AgendaContainer>
  )
}

export default Meet
