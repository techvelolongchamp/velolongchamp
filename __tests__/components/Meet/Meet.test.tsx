import React from 'react'
import { RenderWithProvider as render } from '../../provider'

import Meet from '../../../components/Meet/Meet'

describe('Meet unit tests', () => {
  it('should render correctly', () => {
    const { container } = render(<Meet />)

    expect(container).toMatchSnapshot()
  })
})
