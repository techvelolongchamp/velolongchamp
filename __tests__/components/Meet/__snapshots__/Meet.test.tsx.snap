// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meet unit tests should render correctly 1`] = `
.c0 {
  padding-bottom: 30px;
}

.c0 p {
  font-size: 14px;
  line-height: 20px;
  color: #0d171b;
  padding: 10px 0;
  text-align: justify;
}

.c0 a {
  margin-left: 5px;
  color: #419a1c;
}

.c0 ul {
  list-style: none;
  margin: 0 20px;
}

.c0 ul li {
  color: #6d6e71;
  font-size: 16px;
  font-weight: 500;
  padding: 10px 0;
}

<div>
  <article
    class="c0"
  >
    <p>
      Venez nous rencontrer à l'occasion d'événements informels que nous organiserons.
    </p>
    <p>
      Lieu : à la fontaine de la ligne de départ, en haut du faux plat montant à gauche. Ouvrir la carte : 
      <a
        href="https://goo.gl/maps/2sdy3M1yRUho9k8G9"
        rel="noopener noreferrer"
        target="_blank"
        title="RDV"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 576 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M288 0c-69.59 0-126 56.41-126 126 0 56.26 82.35 158.8 113.9 196.02 6.39 7.54 17.82 7.54 24.2 0C331.65 284.8 414 182.26 414 126 414 56.41 357.59 0 288 0zm0 168c-23.2 0-42-18.8-42-42s18.8-42 42-42 42 18.8 42 42-18.8 42-42 42zM20.12 215.95A32.006 32.006 0 0 0 0 245.66v250.32c0 11.32 11.43 19.06 21.94 14.86L160 448V214.92c-8.84-15.98-16.07-31.54-21.25-46.42L20.12 215.95zM288 359.67c-14.07 0-27.38-6.18-36.51-16.96-19.66-23.2-40.57-49.62-59.49-76.72v182l192 64V266c-18.92 27.09-39.82 53.52-59.49 76.72-9.13 10.77-22.44 16.95-36.51 16.95zm266.06-198.51L416 224v288l139.88-55.95A31.996 31.996 0 0 0 576 426.34V176.02c0-11.32-11.43-19.06-21.94-14.86z"
          />
        </svg>
      </a>
    </p>
    <p>
      Organistation : On partagera boissons et biscuit apéritif ! On essayera de limiter les déchets et peut être que l'on versera la boisson dans vos bidons ! C'est l'occasion de nous présenter.
    </p>
    <p>
      Prochaines dates :
    </p>
    <ul>
      Aucune date prévue pour le moment
    </ul>
  </article>
</div>
`;
